frontend/src/scenes/links/LinkScene.tsx

```diff
@@ -0,0 +1,131 @@
+import { IconCopy, IconDownload } from '@posthog/icons'
+import { LemonButton, LemonDivider, LemonInput, LemonSelect } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { Form } from 'kea-forms'
+import { PageHeader } from 'lib/components/PageHeader'
+import { LemonField } from 'lib/lemon-ui/LemonField'
+import { LemonLabel } from 'lib/lemon-ui/LemonLabel/LemonLabel'
+import { LemonTextArea } from 'lib/lemon-ui/LemonTextArea/LemonTextArea'
+import { QRCodeSVG } from 'qrcode.react'
+import { SceneExport } from 'scenes/sceneTypes'
+
+import { linkConfigurationLogic } from './linkConfigurationLogic'
+
+export function LinkScene({ id }: { id?: string } = {}): JSX.Element {
+    const logic = linkConfigurationLogic({ id: id ?? 'new' })
+    const { link, isLinkSubmitting } = useValues(logic)
+    const { submitLink } = useActions(logic)
+
+    const isNew = id === 'new'
+    const buttonText = isNew ? 'Create link' : 'Update link'
+
+    const fullLink = link?.id
+        ? `${window.location.protocol}//${link.short_link_domain}/${link.short_code}`
+        : 'https://phog.gg/'
+
+    return (
+        <Form
+            id="link"
+            formKey="link"
+            logic={linkConfigurationLogic}
+            props={{ id: id ?? 'new' }}
+            className="deprecated-space-y-4"
+            enableFormOnSubmit
+        >
+            <PageHeader
+                delimited
+                buttons={
+                    <LemonButton type="primary" onClick={submitLink} loading={isLinkSubmitting}>
+                        {buttonText}
+                    </LemonButton>
+                }
+            />
+
+            <div className="space-y-4">
+                <div className="flex gap-8">
+                    <div className="flex-1 space-y-6">
+                        <LemonField name="redirect_url" label="Destination URL">
+                            <LemonInput placeholder="https://posthog.com/links" fullWidth autoWidth={false} />
+                        </LemonField>
+
+                        <div className="flex flex-col gap-2">
+                            <LemonLabel>Link</LemonLabel>
+                            <div className="flex gap-2">
+                                <LemonField name="short_link_domain">
+                                    <LemonSelect
+                                        disabledReason="More domains coming soon"
+                                        options={[
+                                            { label: 'postho.gg', value: 'postho.gg/' },
+                                            { label: 'phog.gg', value: 'phog.gg/' },
+                                            { label: 'hog.gg', value: 'hog.gg/' },
+                                        ]}
```

Comment:
It doesnt make sense to disable the whole select and then include these here. I suggest you make the individual options disabled instead, and make `phog.gg` the only available one